ui/src/widgets/ui-gauge/types/UIGaugeDial.vue

```diff
@@ -316,7 +314,7 @@ export default {
                         const rotate = d3.interpolate(start, end)(t)
                         const deg = rotate * (180 / Math.PI)
                         // -6 is fudge factor to ensure needle is visible and doesn't have half hanging out of SVG window
-                        return `translate(0, ${vue.props.gtype === 'gauge-half' ? vue.sizes.fudge.toString() : '0'}px)rotate(${deg}deg)`
+                        return `translate(0, ${vue.props.gtype === 'gauge-half' ? vue.sizes.fudge.toString() : '0'}px)rotate(${deg}deg)` + needleScale
```

Comment:
<img width="229" alt="Screenshot 2024-09-11 at 12 05 29" src="https://github.com/user-attachments/assets/9bf30a32-7996-4f78-83c5-cdfe525d7ec8">

This seems to cause mis-sizing on the gauge needle at smaller scale...

Reply:
I removed the needle scaling as we can consider iPhone 4/5 as the lowest resolution mobile devices and it won't be a problem

![127 0 0 1_1880_dashboard_page13(iPhone 5_SE)](https://github.com/user-attachments/assets/a9c56d08-a270-435e-944b-4731f959afa2)